packages/core-data/src/actions.js

```diff
@@ -80,7 +80,9 @@ export function receiveEntityRecords(
 ) {
 	// Auto drafts should not have titles, but some plugins rely on them so we can't filter this
 	// on the server.
-	if ( kind === 'postType' ) {
+	// Auto draft templates are automatically converted from files and can have titles,
```

Comment:
It would be very unfortunate if we would have to keep this special handling for `wp_template` post type. Would it be possible to filter this post type on the server to make it fit here?

Reply:
Yeah, I'm not very happy with this as well, but I'm not sure how to do this kind of filtering server-side, when then the title gets wiped out here regardless. 🤔 

Maybe a different approach could be to just simply "embrace" this `auto-draft` quirk of not having titles, and simply always fall back to the default definition when the title is empty. 🤔 
I have already tried something like this and got stuck, but maybe there are different approaches that I could explore, so I'll poke around a bit more.